src/utils.js

```diff
@@ -1326,5 +1326,28 @@ export function cyrb53Hash(str, seed = 0) {
 export function isAllowZeroCpmBidsEnabled(bidderCode) {
   const bidderSettings = getGlobal().bidderSettings;
   return ((bidderSettings[bidderCode] && bidderSettings[bidderCode].allowZeroCpmBids === true) ||
-   (bidderSettings.standard && bidderSettings.standard.allowZeroCpmBids === true));
+    (bidderSettings.standard && bidderSettings.standard.allowZeroCpmBids === true));
+}
+
+/**
+ * Return a memoized version of the given function that uses the first argument as cache key.
+ */
+export function memoize1(fun) {
```

Comment:
Why is this called `memoize1`?  Why not just `memoize`?

Reply:
because it only uses the first argument as the memoization key.